### ISSUE TITLE
RTSP to HTTP?

### ISSUE THREAD

**Issue by iwoloschin** [pvizeli/ha-ffmpeg#11]
Is there a way to use this code to repackage an RTSP MJPEG stream into an HTTP MJPEG stream that HA's frontend can handle?  Ideally I'd like to do this with minimal processing power, mostly because at the moment I'm stuck on a Raspberry Pi 3 which is struggling with repackaging an H.264 stream.

**Comment by pvizeli** [comment]
We can make a copy property, but your mix some things. First you ask for a RTSP MJPEG stream but at end of you message you say you have a H.264 RTSP stream?

**Comment by iwoloschin** [comment]
This issue is sort of resolved by finding an apparently undocumented "full size" HTTP MJPEG Stream, but since it might actually be useful for others, let's keep the dialogue open.

Originally, I thought I could only get video feeds via RTSP (H.264 or MJPEG), which in HomeAssistant requires the ffmpeg component.  Unfortunately, when run on a Raspberry Pi 3 transcoding is too processor intensive, so I was curious if ffmpeg can repackage an RTSP stream into an HTTP stream.  This sounds like if it did work, it'd only be useful for MJPEG streams anyways, not H.264.

**Comment by aequitas** [comment]
Curious about this as well. I also have a ip camera with rtsp and h264 encoding. I've been playing around with the ffmpeg module but thus far I could not get a reliable stream on a raspberry pi, only still images. Copying the h264 stream would probably mean a change in the frontend as well since it only expects mjpegs i guess?

**Comment by pvizeli** [comment]
Yes the poly frontent support only mjpeg. If the frontend Support other things, we can change it here.

**Comment by aequitas** [comment]
Good to know. I currently solved my problem by discovering a JPEG output on my camera, so I won't chase after this further atm.

**Comment by pvizeli** [comment]
I run that with hassio and last ffmpeg on my raspberry with 6 cameras without any problems...

**Comment by aequitas** [comment]
Thanks for the hint. I haven't tried that yet, just compiled libav with hardware acceleration support. Might try it when I consider replacing my current setup.
